test/tools/CodeCoverageAutomation/Start-CodeCoverageRun.ps1

```diff
@@ -1,7 +1,8 @@
 ﻿param(
     [Parameter(Mandatory = $true, Position = 0)] $coverallsToken,
     [Parameter(Mandatory = $true, Position = 1)] $codecovToken,
-    [Parameter(Position = 2)] $azureLogDrive = "L:\"
+    [Parameter(Position = 2)] $azureLogDrive = "L:\",
+    [Parameter] [switch] $SuppressQuiet
```

Comment:
This will make the parameter type of `SuppressQuiet` be [parameter], for example:
```
function bar {
param(
  [Parameter] [switch] $SuppressQuiet
)
}

PS> gcm bar -Syntax
bar [[-SuppressQuiet] <Parameter>]
```